### ISSUE TITLE
highlight(lang, code, ...args) has been deprecated. Please use highlight(code, options) instead.

### ISSUE THREAD

**Issue by vedranmiletic** [marp-team/marp-cli#332]
This message started to appear recently when converting to HTML or PDF via CLI:

```
[  INFO ] Converting 1 markdown...
Deprecated as of 10.7.0. highlight(lang, code, ...args) has been deprecated.
Deprecated as of 10.7.0. Please use highlight(code, options) instead.
https://github.com/highlightjs/highlight.js/issues/2277
Deprecated as of 10.7.0. highlight(lang, code, ...args) has been deprecated.
Deprecated as of 10.7.0. Please use highlight(code, options) instead.
https://github.com/highlightjs/highlight.js/issues/2277
Deprecated as of 10.7.0. highlight(lang, code, ...args) has been deprecated.
Deprecated as of 10.7.0. Please use highlight(code, options) instead.
https://github.com/highlightjs/highlight.js/issues/2277
Deprecated as of 10.7.0. highlight(lang, code, ...args) has been deprecated.
Deprecated as of 10.7.0. Please use highlight(code, options) instead.
https://github.com/highlightjs/highlight.js/issues/2277
Deprecated as of 10.7.0. highlight(lang, code, ...args) has been deprecated.
Deprecated as of 10.7.0. Please use highlight(code, options) instead.
https://github.com/highlightjs/highlight.js/issues/2277
```

**Comment by yhatt** [comment]
Now I'm working on marp-team/marp-core#219 to make it silence.

The warned version of highlight.js should not affect to the output of conversion because it has still keeping compatibility with the classic usage. See also: https://github.com/highlightjs/highlight.js/issues/2277

**Comment by yhatt** [comment]
Fixed by #338.
